projects/coordinator/src/main/java/org/batfish/coordinator/authorizer/FileAuthorizer.java

```diff
@@ -25,17 +26,26 @@
 
    public FileAuthorizer() {
       _logger = Main.getLogger();
-      _usersFile = Main.getSettings().getFileAuthorizerRootDir()
-            .resolve(Main.getSettings().getFileAuthorizerUsersFile());
-      _permsFile = Main.getSettings().getFileAuthorizerRootDir()
-            .resolve(Main.getSettings().getFileAuthorizerPermsFile());
-      if (!Files.exists(_usersFile)) {
-         throw new BatfishException("Users file not found: '"
-               + _usersFile.toAbsolutePath().toString() + "'");
+      try {
+         _usersFile = Main.getSettings().getFileAuthorizerRootDir()
+                 .resolve(Main.getSettings().getFileAuthorizerUsersFile());
+         _permsFile = Main.getSettings().getFileAuthorizerRootDir()
+                 .resolve(Main.getSettings().getFileAuthorizerPermsFile());
+         if (!Files.exists(_usersFile)) {
+            throw new FileNotFoundException("Users file not found: '"
+                    + _usersFile.toAbsolutePath().toString() + "'");
+         }
+         if (!Files.exists(_permsFile)) {
+            throw new FileNotFoundException("Perms file not found: '"
+                    + _permsFile.toAbsolutePath().toString() + "'");
+         }
       }
-      if (!Files.exists(_permsFile)) {
-         throw new BatfishException("Perms file not found: '"
-               + _permsFile.toAbsolutePath().toString() + "'");
+      catch (Exception e) {
+         throw new BatfishException(String.format("Could not initialize FileAuthorizer with "
+                 + " RootDir = %s UsersFile=%s. PermsFile=%s",
+                 Main.getSettings().getFileAuthorizerRootDir(),
+                 Main.getSettings().getFileAuthorizerUsersFile(),
+                 Main.getSettings().getFileAuthorizerPermsFile()));
```

Comment:
Do you want to include the exception `e` in the result? Otherwise, seems weird to throw a FNF with a good message right before it won't get propagated out this catch.